_field-types/mapping-parameters/search-analyzer.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: default
+title: Search analyzer
```

Comment:
There is already one documentation around this https://opensearch.org/docs/latest/analyzers/search-analyzers/

Reply:
@mgodwan yes, there is already one, but we want to add this one here too, I did add a link to the existing one now too, thanks for pointing this out

Reply:
@mgodwan I'm combined the 2 pages into one and left one example in the new page, adding a link to the existing page